UI/Panels/Config/InterpolationPanel.cs

```diff
@@ -49,7 +49,7 @@ internal void syncFromConfig(Interpolation i)
 
             dt.Rows.Clear();
             
-            foreach (float Key in i.GetValues().Keys)
+            foreach (double Key in i.GetValues().Keys)
```

Comment:
`Keys` are defined as a `double` on the Interploation object so update the foreach to use the same data type for `Key`

Reply:
oh! good catch. I wonder if there are other spots where floats are used that used to work when internally a int was used. I will have to search for "float" in the entire project and check